modules/lotamePanoramaIdSystem.js

```diff
@@ -38,16 +38,24 @@ const MISSING_CORE_CONSENT = 111;
 const GVLID = 95;
 const ID_HOST = 'id.crwdcntrl.net';
 const ID_HOST_COOKIELESS = 'c.ltmsphrcl.net';
+const DO_NOT_HONOR_CONFIG = false;
```

Comment:
Could you explain why sometimes we don't want to honor the config?

Reply:
It's in the description for `localStorageIsEnabled` but we only want to honor the config when writing new items. When reading old, or clearing items, we ignore it.